app/src/main/res/layout/activity_main.xml

```diff
@@ -6,23 +6,10 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <Button
-        android:id="@+id/buttonMovies"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:text="open Movies Activity"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <Button
-        android:id="@+id/buttonMovieDetails"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:text="open Movie Details Activity"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/buttonMovies" />
-
+    <FrameLayout
+        android:id="@+id/mainFrame"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
 
+    </FrameLayout>
 </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
here constraint layout not need 
you can use just framelayout or fragmentContainerView in androidX.fragment dependency

Reply:
moved to FragmentContainerView